pelican/log.py

```diff
@@ -92,10 +92,15 @@ class LimitFilter(logging.Filter):
     """
 
     _ignore = set()
+    _raised_messages = set()
     _threshold = 5
     _group_count = defaultdict(int)
 
     def filter(self, record):
+        # don't filter log messages when "debug" is enabled
+        logger_level = logging.getLogger().getEffectiveLevel()
+        if logger_level == logging.DEBUG:
+            return True
```

Comment:
I’m fixed last commit and now it doesn’t interfer with threshold-based filtering. LOG_FILTER works only in production mode and disabled in debug one. It seems that now PR fit both your requirements.